### PR TITLE
Back-end improvements and end-user improvements

### PR DESCRIPTION
**Pull request by Panquesito7**
# Pull Request

## Description

- Align some texts to the center.
- Add favicon to the website. The logo is the one shown on the website's bottom. 
- Add `Nicholas Carrigan` in parentheses after Nicholas' nickname at the bottom.

### CODE REVIEW

**Comment by nhcarrigan** [comment]
Rather than wrapping the h2 elements in p elements, we should use CSS to style the h2 elements directly. 

AFAIK, wrapping headers in paragraph elements isn't semantic HTML.

**Comment by Panquesito7** [comment]
> we should use CSS to style the h2 elements directly

Done. 🙂

File: index.html [hunk 1]
```diff
@@ -19,11 +20,12 @@
...
     <li><a href="#opening">Community</a></li>
-    <li><a href="#becca">Becca</a></li>
-    <li><a href="#whoisNH">About nhcarrigan</a></li>
+    <li><a href="#whoisNH">About Nicholas Carrigan</a></li>
   </ul>
   <div id="welcome">
-    <h1>Welcome to Carrigan's Community! 🔥</h1>
+    <p align="center">
+      <h1>Welcome to Carrigan's Community! 🔥</h1>
+    </p>
```
Comment (nhcarrigan) [inline index.html:28]: This one should also not be within a `p` tag.

File: index.html [hunk 2]
```diff
@@ -19,11 +20,12 @@
     <li><a href="#discussion">Discussions</a></li>
     <li><a href="./views/calendar.html">Events</a></li>
     <li><a href="#opening">Community</a></li>
-    <li><a href="#becca">Becca</a></li>
```
Comment (nhcarrigan) [inline index.html:22]: We should leave this here - I believe we had plans to add a brief section for this.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by Vyvy-vi — COMMENTED** [review]
LGTM!
I'm wondering, if the favicon added here might conflict with the favicon added by #18 ?

**Comment by Panquesito7** [comment]
> might conflict with the favicon added by #18

I'm not so sure. That PR makes a lot of changes and modifies lots of stuff. IMO, it seems too much to simply add a favicon. 🤷‍♂️

**Review by nhcarrigan — APPROVED** [review]
I think I'm good with this.

**Pull request merged**